### PR TITLE
changed behaviour of aggsig on empty lists

### PR DESCRIPTION
**Pull request by matt-o-how**

### CODE REVIEW

File: src/full_node/mempool_manager.py [hunk 1]
```diff
@@ -317,11 +317,23 @@ async def add_spendbundle(
                 continue
 
             # Verify aggregated signature
-            validates = AugSchemeMPL.aggregate_verify(
```
Comment (bramcohen) [inline src/full_node/mempool_manager.py:320]: Why not avoid increasing indentation by inserting the error case up front like this:

if len(msgs) == 0 and new_spend.aggregated_signature != G2Element():
                        return (
                        None,
                        MempoolInclusionStatus.FAILED,
                        Err.BAD_AGGREGATE_SIGNATURE,
                    )

**Review by bramcohen — COMMENTED** [review]
Size of diff could be made smaller

**Comment by mariano54** [comment]
What is the purpose of this change? 

Recently we added infinity elements to blspy so we should use these. In this case, if there are no transactions we will verify the infinity sig with not signatures, which should be true.

I'll make this change in the new consensus branch.

**Comment by hoffmang9** [comment]
Special case “AggSig of nothing” aka “aggregation of an empty list.” Bram sent an email to Riad to see if there was an official way to do that.

**Comment by mariano54** [comment]
Yeah I'm pretty sure validating a signature with no public keys should return false, according to the spec.
So we should make a special case like Bram said, to use the infinity element. There is a way to encode the infinity element, it's coded up in bls-signatures:

>>> blspy.G2Element.infinity()
<G2Element c00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000>

**Comment by mariano54** [comment]
All of this code is being changed in the ms.refactor branch, so no need to change it here

**Comment by hoffmang9** [comment]
I'm closing this as I believe this change has come in via the most recent update of blspy

**Pull request closed without merging**